deployment_scripts/appimage/monkey_island.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.10
+Type=Application
+Name=Monkey Island
+Comment=FILL ME IN
```

Comment:
Fill him in?

Reply:
AppImage Builder doesn't actually use this file, but different packaging methods do. I'll fill this in in round 2 or remove this wile.